### PR TITLE
TMA-904: allow to enable/disable VCR completely by ENV

### PR DESCRIPTION
**Pull request by jimirocks**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/77/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/888/ : SUCCESS in 2m 02s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1156/ : SUCCESS in 2m 43s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/18/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/890/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1158/ : SUCCESS in 38s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1352/ : SUCCESS in 59m 14s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1115/ : UNSTABLE in 31m 10s

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -27,7 +27,9 @@
 require 'pathname'
 require 'webmock/rspec'
 require 'gooddata'
-require 'vcr_configurer'
+
+vcr_enabled = !ENV['VCR_ON'] || ENV['VCR_ON'] == 'true' # VCR is enabled by default - set VCR_ON=false to disable
+require 'vcr_configurer' if vcr_enabled
```
Comment (panjan) [inline spec/spec_helper.rb]: Minor: We could eliminate this extra `if` by moving the `require` to [the `if` block below](https://github.com/gooddata/gooddata-ruby/pull/1168/files#diff-93830fa29d616f7c87903d08b5b1b29aR70).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/23/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/902/ : SUCCESS in 51s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1170/ : SUCCESS in 2m 47s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1357/ : SUCCESS in 56m 50s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1120/ : SUCCESS in 33m 25s

**Pull request merged**